### ISSUE TITLE
Add --quiet or --silent option to tutor

### ISSUE THREAD

**Issue by cacciaresi** [overhangio/tutor#268]
**Is your feature request related to a problem? Please describe.**
Not a problem at all ;) It would be good to have an option to silent the output of tutor related with docker-compose.

**Describe the solution you'd like**
I'd like to have an option (maybe --quiet, --silent, -s) to silent status of containers.

**Additional context**
In our specific case we are running command in a cronjob and the logs are filled with this kind of messages.

See: https://github.com/docker/compose/issues/3447

**Comment by regisb** [comment]
To be clear: the goal is to get rid of the "starting ..." messages.

**Comment by cacciaresi** [comment]
Correct ;)

**Comment by regisb** [comment]
@cacciaresi It occurred to me that you could obtain the same result by running `tutor local start -d > /dev/null`. Right? Would that be a satisfactory solution?

**Comment by cacciaresi** [comment]
When I run it like that, I still see al the messages related with each container status.

```
(tutor) root@open-edx:~# tutor local start -d > /dev/null
Starting tutorlocal_discovery-assets_1 ... 
tutorlocal_memcached_1 is up-to-date
tutorlocal_mongodb_1 is up-to-date
tutorlocal_mysql_1 is up-to-date
tutorlocal_elasticsearch_1 is up-to-date
tutorlocal_smtp_1 is up-to-date
Starting tutorlocal_ecommerce-assets_1 ... 
tutorlocal_rabbitmq_1 is up-to-date
Starting tutorlocal_openedx-assets_1 ... 
tutorlocal_forum_1 is up-to-date
tutorlocal_cms_1 is up-to-date
tutorlocal_lms_1 is up-to-date
tutorlocal_cms_worker_1 is up-to-date
tutorlocal_nginx_1 is up-to-date
tutorlocal_lms_worker_1 is up-to-date
tutorlocal_discovery_1 is up-to-date
tutorlocal_ecommerce_1 is up-to-date
Starting tutorlocal_ecommerce-assets_1 ... done

```

**Comment by regisb** [comment]
Right. It appears that we can silence the docker-compose command by running:

    tutor local start -d 2> /dev/null

But this is not a satisfactory solution, as it swallows actual error messages. We'll figure out something better.

**Comment by regisb** [comment]
This will be addressed by PR #275 once docker/compose#6026 gets fixed.

**Comment by regisb** [comment]
There has been no activity upstream for a long time. I'll close this for now. Please re-open when the folks at docker-compose do something about it.
